composer.json

```diff
@@ -19,10 +19,8 @@
     },
     "require-dev": {
         "doctrine/orm": "~2.5",
-        "phpunit/phpunit": "~4.7",
+        "phpunit/phpunit": "~6.2",
         "doctrine/coding-standard": "dev-master",
-        "mockery/mockery": "^0.9.4",
-        "johnkary/phpunit-speedtrap": "~1.0@dev",
```

Comment:
Because the stable version doesn't support PHPUnit 6. Also I don't see much value on it right now, the only reason we had slow tests on the suite was the fact we were using mockery overload instance feature and that require tests to run in isolated processes, which has been removed.

I can add it back though 😄